src/WebJobs.Script.WebHost/Security/Key.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.CompilerServices;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost
+{
+    public class Key : IEquatable<Key>
+    {
+        public Key()
+        {
+        }
+
+        public Key(string name, string value)
+        {
+            Name = name;
+            Value = value;
+        }
+
+        [JsonProperty(PropertyName = "name")]
+        public string Name { get; set; }
+
+        [JsonProperty(PropertyName = "value")]
+        public string Value { get; set; }
+
+        [JsonProperty(PropertyName = "encrypted")]
+        public bool IsEncrypted { get; set; }
```

Comment:
Wondering if you need the IsEncrypted property. If EncryptionKeyId is set, won't IsEncrypted always be true?